### PR TITLE
Support s01 close out

### PR DESCRIPTION
**Pull request by IsaackMwenda**
This adds support to s01 closeout and updates coda add & get to use master which supports incremental get

### CODE REVIEW

File: configuration/coding_plans.py [hunk 1]
```diff
@@ -108,6 +108,24 @@ def get_rqa_coding_plans(pipeline_name):
                    ws_code=CodeSchemes.WS_CORRECT_DATASET.get_code_with_match_value("OXFAM WASH s01e03 Noise Handler"),
                    raw_field_fold_strategy=FoldStrategies.concatenate),
 
+        CodingPlan(raw_field="s01_close_out_raw",
+                   time_field="sent_on",
+                   run_id_field="s01_close_out_run_id",
+                   coda_filename="OXFAM_WASH_s01_Close_Out.json",
```
Comment (as2388) [inline configuration/coding_plans.py:114]:
```suggestion
                   coda_filename="OXFAM_WASH_s01_close_out.json",
```
(This is the convention we've fallen into for no particular good reason, no need to change anything now though here)

File: configuration/pipeline_config.json [hunk 2]
```diff
@@ -7,14 +7,7 @@
...
       "ContactsFileName": "oxfam_wash_contacts",
       "ActivationFlowNames": [
-        "oxfam_wash_s01e01_activation",
-        "oxfam_wash_s01e02_activation",
-        "oxfam_wash_s01e03_activation",
-        "oxfam_wash_s01e01_beneficiaries_sms_ad",
-        "Oxfam_wash_s01e02_beneficiaries_sms_ad",
-        "oxfam_wash_s01e03_noise_handler",
-        "oxfam_wash_s01_accountability_follow_up",
-        "oxfam_wash_s01_programme_evaluation"
```
Comment (as2388) [inline configuration/pipeline_config.json:17]: Why are these being deleted?

File: run_scripts/1_coda_get.sh [hunk 3]
```diff
@@ -30,7 +31,7 @@ DATASETS=(
 )
 
 cd "$CODA_V2_ROOT/data_tools"
-git checkout "e895887b3abceb63bab672a262d5c1dd73dcad92"  # (master which supports incremental get)
+git checkout "c47977d03f96ba3e97c704c967c755f0f8b666cb"  # (master which supports incremental get)
```
Comment (as2388) [inline run_scripts/1_coda_get.sh]:
```suggestion
git checkout "c47977d03f96ba3e97c704c967c755f0f8b666cb"  # (master which supports incremental add)
```

**Review by as2388 — APPROVED** [review]

File: configuration/coding_plans.py (hunk 1 above)
Comment (IsaackMwenda) [reply configuration/coding_plans.py:114]: Thanks - will keep as-is for consistency with the rest of the coda file names for this project.

File: configuration/pipeline_config.json (hunk 2 above)
Comment (IsaackMwenda) [reply configuration/pipeline_config.json:17]: Thanks for catching this. This was unintended.

**Pull request merged**